README.md

```diff
@@ -52,12 +52,16 @@ After a successful command-line build, the cswinrt.sln can be launched from the
 
 **Note:**  By default, projection projects only generate source files for Release configurations, where cswinrt.exe can execute in seconds.  To generate projection sources for Debug configurations, set the project property GenerateTestProjection to 'true'.  In either case, existing projection sources under the "Generated Files" folder will still be compiled into the projection assembly.  This configuration permits a faster inner loop in Visual Studio.
 
-**Note:** When building the first time, the build clones the testwinrt project. The build depends on this project, but msbuild doesn't consistently pick up the change when the repo is first cloned repo. You may need to build a second time if this is your first build to address the timing issue.
+**Note:** When building the first time, the build clones the testwinrt project. The build depends on this project, but msbuild doesn't consistently pick up the change when the repo is first cloned. You may need to build a second time if this is your first build to address the timing issue.
```

Comment:
"msbuild doesn't consistently ..." 
I haven't seen anything like this.  As long as build.cmd is used, it shouldn't be a problem - was that the issue?

Reply:
ok, let's remove it